web/controllers/password_controller.ex

```diff
@@ -2,36 +2,20 @@ defmodule CodeCorps.PasswordController do
   use CodeCorps.Web, :controller
 
   alias CodeCorps.{Emails, Mailer, User, AuthToken}
-  alias Ecto.Changeset
 
   @doc"""
   forgot_password should take an email and generate an AuthToken model and send an email
   """
   def forgot_password(conn, %{"email" => email}) do
     with %User{} = user <- Repo.get_by(User, email: email),
-        %Changeset{valid?: true} <- AuthToken.changeset(%AuthToken{}, user) do
-
-        try_create_forgot_password(user)
-        |> send_forgot_password_email
-
-        conn
-        |> put_status(200)
-        |> render("show.json", email: user.email)
-
+        { :ok, %AuthToken{} = %{ value: token } } <- AuthToken.changeset(%AuthToken{}, user) |> Repo.insert
```

Comment:
@begedin should I be calling Repo.insert here like this?  Or incorporate that apart of the model?